doc/conf.py

```diff
@@ -100,6 +102,8 @@
 suppress_warnings = ["misc.highlighting_failure"]
 use_jupyterbook_latex = True
 use_multitoc_numbering = True
+# ogp_site_url = "https://jupysql.ploomber.io/en/latest/"
```

Comment:
this line is commented, can we deleted it?

Reply:
we can delete that, for production we have to change ogp_image  to "/en/latest/_static/html-meta-template.png" since  it will take default url https://jupysql.ploomber.io as ogp_site_url

Reply:
ok, please delete this then. once we have a working version for this PR, we can make the necessary changes for this to work in production